packages/compartment-mapper/src/import-hook.js

```diff
@@ -212,6 +222,9 @@ export const makeImportHookMaker = (
       await null;
       compartmentDescriptor.retained = true;
 
+      // for lint rule
```

Comment:
seems odd, there's one above already. is the rule malfunctioning? 

(why is it mal- vs dis- with function?)

Reply:
this is for the lint rule that says every `async` function, if it has an `await`, must have that `await` at the top level.  any `await` in the _closure_ does not satisfy the rule.